alomgir-alom310/week-3/jquery-selector-lab-master/solution/select_fun_part_2_solution/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+    	<meta charset="utf-8">
+    	<meta name="description" content="Your description goes here">
+    	<meta name="keywords" content="one, two, three">
+
+		<title>Blank Template</title>
+
+		<!-- external CSS link -->
+		<link rel="stylesheet" href="css/normalize.css">
+		<link rel="stylesheet" href="css/style.css">
+
+	</head>
+	<body>
+		<div id="container">
+			<header>
+				<h1>Selector Fun</h1>
+				<h2>Slide up the <code>header</code> using jQuery when a user clicks on it</h2>
+			</header>
```

Comment:
I'm not sure what happened in your HTML file for this assignment. Looks like your code is not properly formatted with indentation. I recommend getting into the practice of keeping your code clean and well indented to set good habits. Your code will only get larger and more complex from here.